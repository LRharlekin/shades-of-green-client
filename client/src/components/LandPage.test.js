import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { BrowserRouter as Router, Route } from "react-router-dom";

import { LandPage } from './LandPage';

describe ('LandPage.js', () => {

  afterEach(cleanup);

  it('renders', () => {
    const { asFragment } = render(<div className="LandPageContainer" />)
    expect(asFragment()).toMatchSnapshot();
  });

  it('headline is \'Shades of Green\'', () => {
    const { getByTestId } = render(<Router>  <LandPage/> </Router>);

    expect(getByTestId('h1tag')).toHaveTextContent('Shades of Green');
  })

})