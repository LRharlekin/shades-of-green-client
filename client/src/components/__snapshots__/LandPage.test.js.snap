// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandPage.js renders 1`] = `
<DocumentFragment>
  <div
    class="LandPageContainer"
  />
</DocumentFragment>
`;
